src/blacken_docs/__init__.py

```diff
@@ -244,13 +245,15 @@ def format_file(
         print(f"{filename}:{lineno}: code block parse error {error.exc}")
     if errors and not skip_errors:
         return 2
-    if contents != new_contents:
-        print(f"{filename}: Rewriting...")
-        with open(filename, "w", encoding="UTF-8") as f:
-            f.write(new_contents)
-        return 1
-    else:
+    if contents == new_contents:
         return 0
+    if check_only:
+        print(f"Error: Would have been re-written - {filename}")
```

Comment:
Let’s match the formatting of the existing message.

```suggestion
        print(f"{filename}: Would rewrite.")
```

Reply:
Done in: f01c5ab

I did use a different error message:

```
print(f"{filename}: Requires a rewrite.")
```

I feel like it reads better; but if you're unhappy I can always just do the one you suggest.